### ISSUE TITLE
How do clerks export data?

### ISSUE THREAD

**Issue by waldoj** [CodeforNRV/va-circuit-court-search#1]
I hesitate to approach any clerks, because I know that they're going to say—best case—"OK, sure, I'll export bulk data...but I have absolutely no idea of how to do that." [The sample letter](http://vacircuitcourtsearch.com/sampleLetter) doesn't describe the mechanics of that process, and I certainly don't know anything about it. Do all clerks use the same software? Is the process of exporting data technologically challenging? Can it be automated, or must somebody perform a manual process on a regular schedule?

Describing these specifics is going to be helpful, both to folks who want to lobby clerks and, more important, to the clerks themselves.

**Comment by bschoenfeld** [comment]
Agreed. I'm still working on the sample letter and I hope to address this somehow, but the honest answer is that I don't know. The state's website is managed by the Office of the Executice Secretary. I'm not sure how it works or how they are able to export data. The state does create a number of reports. Here's and example - http://www.courts.state.va.us/courtadmin/aoc/judpln/csi/stats/circuit/cr01annual/cr01_2013.pdf 

I think this would be a good place to start the conversation. We should find out how these reports are generated and then ask if they have the ability to export case details. 

Ideally, the court would allow a bulk download of all cases filled in a particular court for a particular year or month. If it's not possible to generate the download at request time, the batches should be updated at least once every month. 

So the language in the letter needs to be changed to ask for bulk downloads to be added to the existing website. It should also specify a way in which to start a conversation about this if adding bulk downloads is not that simple. 

How does that sound? I'm open to suggestions and help with the rewrite.

**Comment by waldoj** [comment]
I suspect, then, that it's too early to start asking clerks to take action, if there's no known action for the clerks to take. I think the average clerk looks at this, thinks "I have absolutely no concept of where to start with this," and benignly ignoring it. Perhaps what would be really useful is IDing just one clerk who is particularly technologically savvy, and finding out from that person what the possibilities are? That might just be a matter of somebody (perhaps Dave Ress) asking a friendly clerk "hey, which clerk in Virginia is really good with tech?" and then talking to that person. I'll ask a clerk friend and see if she can provide a pointer.

**Comment by bschoenfeld** [comment]
That might be a good start. I've been meaning to reach out to my local clerk. Dave has had luck with getting a directive from the local clerk for the Office of the Executive Secretary to give Dave access to the data. Dave doesn't know what form that access will come in.

Previously, Dave engaged the Office of the Executive Secretary and didn't get a warm reception. So the hope with this effort is actually just to get permission from the local clerk to access the data in bulk. Once we get that permission from enough clerks, we are hoping to force a conversation at the state level. What do you think of that strategy?

**Comment by waldoj** [comment]
It's a fine strategy, certainly. If we can't get the data from the top, we'll siphon it from 130-odd other spigots. :) I quite like the directive idea! It seems like, if that's working for Dave, that we might try this plan:
- Assemble a list of every court, their clerks, and the clerk's e-mail address. (Assuming that they are responsive to e-mail. I have no experience with this.)
- Create a stock letter that's comprised of two parts—a letter to the clerk, making a request, and a directive that's ready for the clerk to e-mail (again, assuming e-mail is acceptable) to the Office of the Executive Secretary, granting access to the data to a central person (e.g., you).
- Create a tracker in this repository where anybody can claim a given court as being within their responsibility, meaning that they're going to send a request and track the outcome. I suspect that an issue should be opened for each request, because some of those requests are going to become complicated.

How does that sound?

**Comment by bschoenfeld** [comment]
I love it. I'll draft something up and submit for feedback.
